nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -1422,10 +1424,15 @@ public String getDefaultBackPressureDataSizeThreshold() {
     public static NiFiProperties createBasicNiFiProperties(final String propertiesFilePath, final Map<String, String> additionalProperties) {
         final Map<String, String> addProps = (additionalProperties == null) ? Collections.EMPTY_MAP : additionalProperties;
         final Properties properties = new Properties();
-        final String nfPropertiesFilePath = (propertiesFilePath == null)
+        String nfPropertiesFilePath = (propertiesFilePath == null)
                 ? System.getProperty(NiFiProperties.PROPERTIES_FILE_PATH)
                 : propertiesFilePath;
         if (nfPropertiesFilePath != null) {
+            try {
```

Comment:
Was this included on accident? Does not seem relevant to your ticket.

Reply:
it was relevant, to get it working in my local environment, where someone had put spaces in the path, %20 in resource paths needed converting back to ' '

Reply:
This sounds like it could be a legitimate bug, but it would need to be submitted separately as it's not related to fixing `GetFile`. I run on Windows, but I don't have spaces in the path, maybe switch to a new path with no spaces for now?